packages/search-in-workspace/src/browser/search-in-workspace-service.ts

```diff
@@ -112,8 +112,7 @@ export class SearchInWorkspaceService implements SearchInWorkspaceClient {
             throw new Error('Search failed: no workspace root.');
         }
 
-        const rootUri = new URI(root.uri);
-        const searchId = await this.searchServer.search(what, rootUri.path.toString(), opts);
+        const searchId = await this.searchServer.search(what, FileUri.fsPath(root.uri), opts);
```

Comment:
Is there an issue with the path handling in the search service?  Should we move this conversion to the service implementation?

Could you point out, what's the real issue with given path on win?

Reply:
`fsPath` has some special handling for windows: https://github.com/Microsoft/vscode-uri/blob/b1d3221579f97f28a839b6f996d76fc45e9964d8/src/index.ts#L955-L976

We can pass a URI instead and call `FileUri.fsPath` on the backend.

Reply:
@AlexTugarev, on Windows the search service passes a path with a leading slash to vscode-ripgrep. So searches always returns empty.  FileUri.fsPath() fixes this.

Reply:
@gonzalezw, we should not use `FileUri` in the browser code. We had a very [similar issue with the file search](https://github.com/theia-ide/theia/issues/1584) (<kbd>Ctrl/Cmd</kbd>+<kbd>P</kbd>). Please look into [that PR](https://github.com/theia-ide/theia/pull/1587). Perhaps this is something we have to fix on the backend.